pkg/hostagent/setup.go

```diff
@@ -284,6 +286,16 @@ func (agent *HostAgent) configureContainerIfaces(metadata *md.ContainerMetadata)
 			}
 		}
 
+		for _, ip := range iface.IPs {
```

Comment:
If we had a dual-stack setup, this would create two veths

Reply:
for the dual-stack ips, the code will exit the loop if  iface.HostVethName and iface.Mac is already assigned